README.md

```diff
@@ -140,3 +257,23 @@ In order to run the tests in random mode, the target Open Booking API implementa
 # Contributing
 
 - [Contributing to the project](./CONTRIBUTING.md)
+
+# Concepts
+
+## Booking Partner Authentication Strategy
```

Comment:
This link is used in this GitBook PR (in draft at time of writing): https://app.gitbook.com/o/-LJTExWaXZFqcFJ_lZwF/s/-LOMz1IIOd5fl4xzfUYb/~/changes/273/open-booking-api/key-decisions